examples/misc/anchored_artists.py

```diff
@@ -56,48 +69,57 @@ def __init__(self, transform, width, height, angle, loc,
                          child=self._box, prop=prop, frameon=frameon)
 
 
-class AnchoredDrawingArea(AnchoredOffsetbox):
-    def __init__(self, width, height, xdescent, ydescent,
-                 loc, pad=0.4, borderpad=0.5, prop=None, frameon=True):
-        self.da = DrawingArea(width, height, xdescent, ydescent)
-        super().__init__(loc, pad=pad, borderpad=borderpad,
-                         child=self.da, prop=None, frameon=frameon)
+def draw_ellipse(ax):
+    """
+    Draw an ellipse of width=0.1, height=0.15 in data coordinates
+    """
+    ae = AnchoredEllipse(ax.transData, width=0.1, height=0.15, angle=0.,
+                         loc=3, pad=0.5, borderpad=0.4, frameon=True)
 
+    ax.add_artist(ae)
 
-if __name__ == "__main__":
```

Comment:
Well, OK. I did delete this line, and made the same set of small functions as in the similar file to highlight the parallels.